src/main/java/lk/gov/health/phsp/bean/ClientController.java

```diff
@@ -639,31 +641,23 @@ public void saveAllImports() {
 //        }
 //        return true;
 //    }
+    
     public String importClientsFromExcel() {
```

Comment:
Did you check this method is working after the code change? I think it is essential to check this method again before merging.

Reply:
I added a task (https://dinidu.atlassian.net/browse/HIMS-208) to check because I don't have a sample file for that. If you have one please send me otherwise we can discuss that in today's meeting. until that we can remain this as unmerge. I totally agree with your decision.